package.json

```
@@ -77,14 +78,12 @@
     "prop-types": "15.8.1",
     "querystring-browser": "^1.0.4",
     "react": "=17.0.2",
-    "react-ace": "^4.1.6",
-    "react-addons-css-transition-group": "^15.4.2",
+    "react-ace": "^10.1.0",
```

SwaggerEditor will throw runtime errors and will not render with react-ace@10. What we can do is to update to `react-ace@7.0.5` as in this version there are no breaking changes. Any version above will require code changes and will break integrations and plugins of 3rd parties integrating with SwaggerEditor and expecting react-ace compatible API.

```suggestion
    "react-ace": "=7.0.5",
```